erpnext/assets/doctype/location/location.js

```diff
@@ -11,7 +11,14 @@ frappe.ui.form.on('Location', {
 			};
 		});
 	},
-
+	refresh: function (frm) {
```

Comment:
Why is this needed?
`All Locations` is just a label node displayed on top and each group node under `All Locations` would not have parent node and will be regarded as `root_node` which is kinda wrong. There should only be one root node.